.github/workflows/deploy.yml

```
@@ -78,7 +81,11 @@ jobs:
     runs-on: ubuntu-latest
     needs: [development, build]
     # only build/deploy main branch on pushes
-    if: ${{ (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/dev') && github.event_name == 'push' }}
+    if: ${{ ((github.ref == 'refs/heads/main' || github.ref == 'refs/heads/dev') && github.event_name == 'push') || (github.event_name == 'pull_request' && github.event.label.name == 'staging') }}
```

Let's remove the check for `dev` branch, we're not using that.  Just leftover from the Remix template.

```suggestion
    if: ${{ (github.ref == 'refs/heads/main' && github.event_name == 'push') || (github.event_name == 'pull_request' && github.event.label.name == 'staging') }}
```